lib/cli/parse.js

```diff
@@ -39,6 +37,7 @@ const parser = require('yargs')('run --help')
       type: 'array',
     },
   })
+  .strict()
```

Comment:
This is a nice addition. I had actually forgotten the API and had been using `-q` for "quiet" for quite a while now 🤦‍♂